readme.md

```diff
@@ -6,15 +6,13 @@ This package contains a custom render and a set of functions that can be used wi
 ```
 yarn add @kaliber/elasticsearch
 ```
-
+Note: This library is precompiled using kaliberjs/pack (https://github.com/kaliberjs/pack). Compiling it a second time with will cause this library to stop working.
```

Comment:
```suggestion
> **Note**
This library is precompiled using [kaliberjs/pack](https://github.com/kaliberjs/pack). Compiling it a second time with will cause this library to stop working. This means that you should **not** add it to the `compileWithBabel` array of @kaliber/build
```

Reply:
https://twitter.com/diegohaz/status/1527642881384759297

Reply:
Deze library kan in principe ook met andere build systems gebruikt worden, en staat ook niet in een private repo. Dus is de comment over @kaliber/build niet iets te specifiek?

Reply:
Misschien `For example` toevoegen aan de zin. Ik denk eerlijk gezegd niet dat er mensen buiten kaliber om deze package snel zullen gebruiken, maar het wel een goeie hint is voor mensen bij ons intern.